tensorboard/data/server/main.rs

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+use byteorder::{ByteOrder, LittleEndian};
+
+fn main() {
+    let ptr = LittleEndian::read_u32(b"\x2e\x68\x63\x73"); // look, a dependency!
```

Comment:
Oh, just <https://xkcd.com/138/>. Nothing germane at all. :-)

Yes, it is rare indeed for normal Rust code to deal with raw pointers
(`*const T` and `*mut T`) rather than references (`&T` and `&mut T`).